authors/settings.py

```diff
@@ -79,10 +80,15 @@
 # Database
 # https://docs.djangoproject.com/en/1.11/ref/settings/#databases
 
+#Check the env.example file to get the idea of how your .env will be structured
 DATABASES = {
     'default': {
-        'ENGINE': 'django.db.backends.sqlite3',
-        'NAME': os.path.join(BASE_DIR, 'db.sqlite3'),
+        'ENGINE': 'django.db.backends.postgresql',
+        'NAME': getenv('DEV_DATABASE_NAME'), # Your databse name from the .env file
+        'USER': getenv('DEV_DATABASE_USER'), # Your postgresql username from the .env file
+        'PASSWORD': getenv('DEV_DATABASE_PASSWORD'), # Your postgresql password from the .env file
+        'HOST': '127.0.0.1', # Localhost for develop
+        'PORT': '', # Postgresql port
     }
```

Comment:
Having decided to use environment variables you can clean this up by having all these details loaded from a dictionary in the .env file. And you do not need to have the in line comments